test/spec/differ.js

```diff
@@ -357,20 +380,20 @@ describe('diffing', function() {
         // then
         expect(results._added).to.have.keys([
           'ManualTask_1',
-          'ExclusiveGateway_1'
+          'ExclusiveGateway_1',
+          'undefined'
         ]);
 
         expect(results._removed).to.have.keys([
-          '_6-674', '_6-691', '_6-746', '_6-748', '_6-74', '_6-125', '_6-178', '_6-642'
+          '_6-674', '_6-691', '_6-746', '_6-748', '_6-74', '_6-125', '_6-178', '_6-642', 'undefined'
         ]);
 
         expect(results._layoutChanged).to.have.keys([
           '_6-61'
         ]);
 
         expect(results._changed).to.have.keys([
-          '_6-127',
-          '_6-219'
+          '_6-127'
```

Comment:
This seems to be a regression.

Old
```xml
    <semantic:intermediateCatchEvent id="_6-219" name="60 minutes">
      <semantic:incoming>_6-424</semantic:incoming>
      <semantic:outgoing>_6-426</semantic:outgoing>
      <semantic:timerEventDefinition>
        <semantic:timeDate xsi:type="semantic:tFormalExpression"/>
      </semantic:timerEventDefinition>
    </semantic:intermediateCatchEvent>
```

new
```xml
        <semantic:intermediateCatchEvent parallelMultiple="false" name="60 minutes" id="_6-219">
            <semantic:incoming>_6-424</semantic:incoming>
            <semantic:outgoing>_6-426</semantic:outgoing>
            <semantic:timerEventDefinition>
                <semantic:timeDate/>
            </semantic:timerEventDefinition>
        </semantic:intermediateCatchEvent>
```

Parallell multiple change should have been caught, but might be removed given that it has a timer element within it. Will investigate whi